fhir-server/liberty-config/config/default/fhir-server-config-db2.json

```diff
@@ -117,30 +117,83 @@
             }
         },
         "bulkdata": {
-            "bulkDataBatchJobIdEncryptionKey": "change-password",
-            "applicationName": "fhir-bulkimportexport-webapp",
-            "moduleName": "fhir-bulkimportexport.war",
-            "jobParameters": {
-                "cos.bucket.name": "fhirbulkdata",
-                "cos.location": "us",
-                "cos.endpoint.internal": "https://minio:9000",
-                "cos.endpoint.external": "https://minio:9000",
-                "cos.credential.ibm": "N",
-                "cos.api.key": "minio",
-                "cos.srvinst.id": "change-password"
+            "enabled": true,
+            "core": {
+                "api": {
+                    "url": "https://localhost:9443/ibm/api/batch",
+                    "user": "fhiradmin",
+                    "password": "change-password",
+                    "truststore": "resources/security/fhirTrustStore.p12",
+                    "truststore-password": "change-password", 
+                    "trust-all": true
```

Comment:
what does this do?

Reply:
sets the client ssl to trust the ca and hostname in the http client used in httpwrapper